### PR TITLE
delete clipnorm

### PR DESCRIPTION
**Pull request by luliyucoordinate**
delete clipnorm

### CODE REVIEW

**Comment by hgaiser** [comment]
Could you merge all PRs in one PR? They seem highly correlated.

Also, why should we delete this?

**Comment by luliyucoordinate** [comment]
Sorry l don't know who to merge all in Browser interface.
The reason for deleting it is not stated in keras Documentation
```
Adam
keras.optimizers.Adam(lr=0.001, beta_1=0.9, beta_2=0.999, epsilon=None, decay=0.0, amsgrad=False)
Adam optimizer.

Default parameters follow those provided in the original paper.

Arguments

lr: float >= 0. Learning rate.
beta_1: float, 0 < beta < 1. Generally close to 1.
beta_2: float, 0 < beta < 1. Generally close to 1.
epsilon: float >= 0. Fuzz factor. If None, defaults to K.epsilon().
decay: float >= 0. Learning rate decay over each update.
amsgrad: boolean. Whether to apply the AMSGrad variant of this algorithm from the paper "On the Convergence of Adam and Beyond".
```

**Comment by hgaiser** [comment]
https://github.com/keras-team/keras/blob/master/keras/optimizers.py#L57

**Comment by luliyucoordinate** [comment]
But when i run code , i find this `keras_retinanet/bin/train.py:101: UserWarning: Output "clipped_boxes" missing from loss dictionary. We assume this was done on purpose, and we will not be expecting any data to be passed to "clipped_boxes" during training.
  optimizer=keras.optimizers.adam()`

**Comment by hgaiser** [comment]
Ah but I assume the warning stayed even after removing the clipnorm argument? The warning and the argument are unrelated.

Also: https://github.com/fizyr/keras-retinanet/blob/master/README.md#notes

**Comment by hgaiser** [comment]
Closing this since I don't think we should delete this.
